src/main/kotlin/nl/info/zac/authentication/UserPrincipalFilter.kt

```diff
@@ -50,12 +50,25 @@ constructor(
         private const val GROUP_MEMBERSHIP_CLAIM_NAME = "group_membership"
     }
 
+    private val adminUriPrefixes = listOf(
+        "/admin/",
```

Comment:
This URI is not for any of our endpoints but for frontend admin URIs only. We don't have integration tests for this (obviously; because our integration tests do not test the frontend at all) and I'm not sure this will work in such a filter? You would need to test manually. The idea here was that any non-beheerder user should not be able to access any admin frontend page. E.g. https://zaakafhandelcomponent-zac-dev.dimpact.lifely.nl/admin/check